frontend/src/pages/modelRegistry/screens/RegisteredModelsArchive/RegisteredModelsArchiveListView.tsx

```diff
@@ -68,7 +69,10 @@ const RegisteredModelsArchiveListView: React.FC<RegisteredModelsArchiveListViewP
                   }))}
                   value={searchType}
                   onChange={(newSearchType) => {
-                    setSearchType(newSearchType as SearchType);
+                    const newSearchTypeInput = asEnumMember(newSearchType, SearchType);
+                    if (newSearchTypeInput !== null) {
+                      setSearchType(newSearchTypeInput);
+                    }
```

Comment:
Can you explain to me why this is an error that required fixing while in `RegisteredModelListView` it wasn't?

Reply:
I must have missed the null check